swan-lake/development-tutorials/source-code-dependencies/style-guide/coding-conventions.md

```diff
@@ -188,7 +188,7 @@ public function main() {
 
 ```ballerina
 int[] a = [];
-int|() result = getResult();
+int? result = getResult();
```

Comment:
Is this correct? Looks like this is specifically an example for parentheses. If so, we can keep the previous version here.

Reply:
Updated.